queries/task_queries/Task_4.2.sql

```diff
@@ -0,0 +1,7 @@
+﻿SELECT [bank-task].dbo.Social_Status.Status_name,
+       (SELECT COUNT(*)
+        FROM [bank-task].dbo.Cards, [bank-task].dbo.Accounts, [bank-task].dbo.Clients
+        WHERE Cards.Account_id = Accounts.AccountID
+          AND Accounts.Client_id = Clients.ClientID
+          AND Clients.Status_id = [bank-task].dbo.Social_Status.StatusID) AS CardsCount
```

Comment:
![image](https://user-images.githubusercontent.com/67163370/189296309-7a8c767b-0229-4f8f-9edc-a29ae7fa6c73.png)

Result must be same, but have different naming in columns